.github/workflows/build_test.yml

```
@@ -92,7 +92,7 @@ jobs:
     runs-on: windows-latest
     
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v4.2.2
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)